src/main/resources/schemas/schema-form.json

```diff
@@ -27,10 +27,22 @@
       "description": "User claim field used to store end user on log analytics",
       "type": "string",
       "default": "sub"
+    },
+    "verifyHost" : {
+      "title": "Verify host",
+      "type" : "boolean",
+      "default": false
+    },
+    "trustAll" : {
+      "title": "Trust all",
+      "type" : "boolean",
```

Comment:
A description for users would be welcomed here.

Reply:
fixed